src/components/Network/NodeSettings/NodeSettings.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import NodeSettingsForm from './NodeSettingsForm'
+
+export default class NodeSettings extends Component {
+  static propTypes = {
+    isCheckingForUpdate: PropTypes.bool,
+    status: PropTypes.shape({
+      client: PropTypes.string,
+      binPath: PropTypes.string,
+      version: PropTypes.string,
+      commit: PropTypes.string,
+      architecture: PropTypes.string,
+      go: PropTypes.string,
+      isRunning: PropTypes.bool
+    }),
+    config: PropTypes.shape({
+      name: PropTypes.string,
+      dataDir: PropTypes.string,
+      host: PropTypes.string,
+      port: PropTypes.number,
+      network: PropTypes.string
+    }),
+    options: PropTypes.shape({
+      syncModes: PropTypes.arrayOf(PropTypes.string),
+      versions: PropTypes.arrayOf(PropTypes.string),
+      ipcOptions: PropTypes.arrayOf(PropTypes.string),
+      networks: PropTypes.arrayOf(PropTypes.string)
+    }),
+    updateStatus: PropTypes.func,
+    changeConfig: PropTypes.func,
+    checkUpdate: PropTypes.func,
+    onStartStop: PropTypes.func
+  }
+
+  render() {
+    const {
+      status,
+      config,
+      options,
+      isCheckingForUpdate,
+      checkUpdate,
+      onStartStop,
+      changeConfig,
+      updateStatus
+    } = this.props
+    const { binPath, client } = status
+
+    return (
+      <StyledSettings>
+        <StyledH1>Node Settings</StyledH1>
+        <StyledH3>
+          Node: <StyledValue>{client}</StyledValue>
+        </StyledH3>
+        <StyledH3>
+          Path: <StyledValue>{binPath}</StyledValue>
+        </StyledH3>
```

Comment:
Unsure if this merits a component in the lib; its heavy with props, but all it does is add a couple headers and pass the rest to the form. Does it make sense for this to just be implemented in the mist-ui?

Reply:
Yeah it's simple for now but I think it is a useful wrapper container for the future that can be added to and customized.
Its original intention was to hold the "templates" concept that philipp had, which we can add later